README.md

```diff
@@ -467,14 +467,32 @@ With the use of the [Home Assistant RESTful][ha-rest] integration, one should
 be able to grab some interesting data from this add-on.
 
 Example:
-
+Set the port in the add-on settings; for example: 6789 and your IP of your Home Assistant instance; the hassio.local didn't work for me.
+Add a rest sensor:
 ```yaml
-sensor:
-  - platform: rest
-    resource: http://a0d7b954-wireguard
+platform: rest
+name: wireguard_sensor
+resource: http://192.168.3.72:6789/
+value_template: 'TEST'
```

Comment:
Maybe we could use the number of peers?

Reply:
That would be good; is there a way to get connected peers?

Reply:
I believe the max handshake time for a client is 180 seconds, latest_handshake is an epoch, so you could convert it and assume connected if less than that?

Reply:
latest_handshake has been 0 for me all this time.

Reply:
Actually, you cannot determine if the client is connected or not. There is no such thing.
However, if the keep-alive has been set, you could be able to check if that has passed or not... 🤔 

Maybe we should add the set keep-alive to the json output.

Reply:
No there is no such thing in WireGuard actually.

Reply:
Well it is doable..

This shows the number of peers that was last heard from in the last **180 s**
```suggestion
value_template: "{% set count = namespace(value=0) %} 
{%- for peer in value_json %}
{%- if ((now() | as_timestamp()) - value_json[peer]['latest_handshake']) < 180 %}
{%- set count.value = count.value + 1 %}{% endif %}{% endfor %}{{ count.value }}"
```

Reply:
There is a `PersistentKeepalive` option but it is not encouraged to be used.

 https://www.wireguard.com/quickstart/#nat-and-firewall-traversal-persistence